.ci/azure-pipelines/env.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       Ubuntu 18.04:
         # Test the oldest supported version of Ubuntu
         UBUNTU_VERSION: 18.04
-        VTK_VERSION: 7
+        VTK_VERSION: 6
```

Comment:
Please don't revert your earlier PR, leave it at `7`

Reply:
Done